Frameworks/SafariServices/SFSafariWebViewController.mm

```diff
@@ -177,4 +207,8 @@ - (void)_cancel {
     [self dismissViewControllerAnimated:NO completion:nil];
 }
 
+- (void)dealloc {
+    _webView.delegate = nil;
```

Comment:
Since we own our webview, doesn't it not matter that its delegate is being nil'd out? It won't have a word to say about it in just a few more seconds.